src/index.html

```diff
@@ -11,7 +11,139 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+    action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    onsubmit="onSubmit()" method="POST"
+    >
+
+      <fieldset>
+        <legend>Personal information</legend>
+        <div class="form-field">
+          <label for="Surname">Surname:</label>
```

Comment:
```suggestion
          <label for="Surname" class="form-field">Surname:</label>
```
You can get rid of this `div` wrappers and give the class to <label> and give it display:block